lib/spack/spack/test/modules/common.py

```diff
@@ -48,7 +46,7 @@ def mock_package_perms(monkeypatch):
 
     yield perms
 
-
+@pytest.mark.skipif(sys.platform == "win32", reason="Symlinks do not work on windows yet")
```

Comment:
Symlinks do work on Windows now, is there a specific reason they don't work here?

Reply:
Ok great, that's actually something I'm working on with the whole chmod/file permissions thing. Can you add a `todo:johnwparent` comment above this?